doc/src/compute_gyration_shape_chunk.txt

```diff
@@ -0,0 +1,88 @@
+three shape parameters of multiple chunks of atoms. The computation includes 
+all effects due to atoms passing through periodic boundaries.
+
+The three computed shape parameters are the asphericity, b, the acylindricity, c,
+and the relative shape anisotropy, k:
+
+:c,image(Eqs/compute_shape_parameters.jpg)
+
+where lx <= ly <= lz are the three eigenvalues of the gyration tensor.
+The asphericity  is always non-negative and zero only when the three principal
+moments are equal. This zero condition is met when the distribution of particles
+is spherically symmetric (hence the name asphericity) but also whenever the particle
+distribution is symmetric with respect to the three coordinate axes, e.g.,
+when the particles are distributed uniformly on a cube, tetrahedron or other Platonic
+solid. The acylindricity is always non-negative and zero only when the two principal
+moments are equal. This zero condition is met when the distribution of particles is
+cylindrically symmetric (hence the name, acylindricity), but also whenever the particle
+distribution is symmetric with respect to the two coordinate axes, e.g., when the
+particles are distributed uniformly on a regular prism. the relative shape anisotropy
+is bounded between zero (if all points are spherically symmetric) and one
+(if all points lie on a line).
+
+The tensor keyword must be specified in the compute gyration/chunk command.
+
+NOTE: The coordinates of an atom contribute to the gyration tensor in
+"unwrapped" form, by using the image flags associated with each atom.
+See the "dump custom"_dump.html command for a discussion of "unwrapped"
+coordinates. See the Atoms section of the "read_data"_read_data.html
+command for a discussion of image flags and how they are set for each
+atom.  You can reset the image flags (e.g. to 0) before invoking this
+compute by using the "set image"_set.html command.
+
+[Output info:]
+
+This compute calculates a global array with six columns, 
+which can be accessed by indices 1-6. The first three columns are the
+eigenvalues of the gyration tensor followed by the asphericity, the acylindricity
+and the relative shape anisotropy.  The computed values can be used by any command
+that uses global array values from a compute as input.  See the "Howto
+output"_Howto_output.html doc page for an overview of LAMMPS output
+options.
+
+The array calculated by this compute is
+"intensive".  The first five columns will be in
+distance^2 "units"_units.html while the sixth one is dimensionless.
+
+[Restrictions:]
+
+This compute is part of the USER-MISC package.  It is only enabled if
+LAMMPS was built with that package.  See the "Build
+package"_Build_package.html doc page for more info.
+
+[Related commands:]
+
+"compute gyration/chunk"_compute_gyration_chunk.html
+"compute gyration/shape"_compute_gyration_shape.html
+
+[Default:] none
+
+:line
+
+[(Theodorou)] Theodorou, Suter, Macromolecules, 18, 1206 (1985).
```

Comment:
The change made on these 2 citations is incorrect in the parent file (compute gyration shape.txt)
and here.  The link(Name) tag needs to be listed.  And it needs to be listed
somewhere in the text above.  Could be as simple as a  sentence like:  This method is described in
Theodoru and Mattice.  (see other examples in the doc pages for the syntax to do this)